db/seeds.rb

```diff
@@ -20,4 +20,77 @@
                email: email,
                password:              password,
                password_confirmation: password)
-end
\ No newline at end of file
+end
+
+
+# Make a rhizome
+Rhizome.create!(
+  name: "Test rhizome"
+  )
+
+ParticleDevice.create!(
+  device_id: Faker::Lorem.characters(24),
+  encrypted_access_token: "S3jF/ramSAL8dZ4zzASXeuBunWv6tLn9jzcU4ABEflfGQkrwbM44CpWMtyRhkED/",
+  rhizome_id: 1
+  )
+
+
+# Make an equipment profile
+EquipmentProfile.create!(
+  name: "Test EP")
+
+HeatingElement.create!(
+  pins: {"power_pin": "1", "control_pin": "0"},
```

Comment:
Rather than passing in a hash to `:pins`, just leverage the store_accessor and use the pins directly. i.e. 
```ruby
...create!(
  power_pin: 1,
  control_pin: 0,
  EquipmentProfile.first
)
```
Do that for all of these Equipment subclass seeds.